darts/models/forecasting/ensemble_model.py

```diff
@@ -229,3 +257,26 @@ def find_max_lag_or_none(lag_id, aggregator) -> Optional[int]:
 
     def _is_probabilistic(self) -> bool:
         return all([model._is_probabilistic() for model in self.models])
+
+    def _full_past_covariates_support(self) -> bool:
+        return all([model.supports_past_covariates for model in self.models])
+
+    def _full_future_covariates_support(self) -> bool:
+        return all([model.supports_future_covariates for model in self.models])
+
+    def _logger_info_covariates_handling(
+        self,
+        past_covariates: Union[TimeSeries, Sequence[TimeSeries]],
+        future_covariates: Union[TimeSeries, Sequence[TimeSeries]],
+    ):
+        if past_covariates is not None and not self._full_past_covariates_support():
```

Comment:
Should we just improve the documentation of the models instead of logging the info?
I'm always a bit careful with verbosity in fit/predict due to historical_forecasting spamming with prints :D 
Or at least avoid that it gets printed infinitely.

Reply:
Added the docstring about past/future covariates usage to the `NaiveEnsemble` and `RegressionEnsemble` to make it more visible in the doc.
I also moved the logger messages to the constructor to avoid spam in historical forecast (left it as `info` to avoid an irrelevant warning for users not using past/future covariates)